supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -205,7 +205,11 @@ When combined with privileges, permissions define a user's or resource's overall
 [discrete]
 [[pid]]
 ==== image:images/yes.png[yes] PID (noun)
+<<<<<<< HEAD
```

Comment:
Looks like there's some leftover text from resolving a merge conflict. Should be removed, correct?